tests/util/test_timeout.py

```diff
@@ -338,3 +338,24 @@ async def test_simple_zone_timeout_zone_with_timeout_exeption() -> None:
                     raise RuntimeError
 
             await asyncio.sleep(0.3)
+
+
+async def test_multiple_global_freezes(hass: HomeAssistant) -> None:
+    """Test multiple global freezes."""
+    timeout = TimeoutManager()
+
+    async def background(delay: float) -> None:
+        async with timeout.async_freeze():
+            await asyncio.sleep(delay)
+
+    async with timeout.async_timeout(0.1):
```

Comment:
There's no need for this or any other timeout util context manager to be async, should we, in a separate PR, convert it to a regular context manager?